### DISCUSSION TITLE
Proxy response validation doesn't fail with values that aren't in specification

### DISCUSSION THREAD

**Discussion by sromeu-ob** [stoplightio/prism#2251] [General]
Hi,
I think this is a bug but I wanted to ask to ensure...

I am trying de proxy validation and all seems to work fine except one thing:

- Prism doesn't get an error when the response returns a field that isn't in the specification.

It should return an error, right? Or am I missing something?

**Comment by saturna** [comment]
Try setting `additionalProperties: false` on the schema object.
